backend/transcribee_backend/auth.py

```diff
@@ -66,11 +65,19 @@ def validate_user_authorization(session: Session, authorization: str):
     raise HTTPException(status_code=401)
 
 
-def get_user_token(
-    authorization: str = Header(),
-    session: Session = Depends(get_session),
-):
-    return validate_user_authorization(session, authorization)
+class TokenAuth(HTTPBase):
+    def __init__(self):
+        self.model = HTTPBaseModel(scheme="token")
+        self.scheme_name = "Token"
+
+    async def __call__(
+        self, request: Request, session: Session = Depends(get_session)
+    ) -> Optional[UserToken]:
```

Comment:
Is this really `Optional`? I think we always return a `UserToken`, otherwise we raise an Exception